src/jdk.jdi/share/classes/com/sun/tools/example/debug/tty/EventHandler.java

```diff
@@ -118,6 +123,10 @@ private boolean handleEvent(Event event) {
             return threadDeathEvent(event);
         } else if (event instanceof VMStartEvent) {
             return vmStartEvent(event);
+/*[IF CRIU_SUPPORT]*/
+        } else if (event instanceof VMRestoreEvent) {
+            return vmRestoreEvent(event);
```

Comment:
Suggest the cast should be here (like later versions) rather than within `vmRestoreEvent()`.

Reply:
Updated